src/PictureCard/README.md

```diff
@@ -0,0 +1,77 @@
+# PictureCard
+
+To implement PictureCard component into your project you'll need to add the import:
+
+```jsx
+import PictureCard from "@kiwicom/orbit-components/lib/PictureCard";
+```
+
+After adding import into your project you can use it simply like:
+
+```jsx
+<PictureCard
+  image={{
+    original: "385x320",
+    code: "dubai_ae",
+    name: "dubai_ae",
```

Comment:
Might be better to use "Dubai Arab Emirates"  in `name` so we don't  give a bad message, that `code` and `name` should be the same in content.